subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeErrorIntegrationTest.groovy

```diff
@@ -25,7 +25,7 @@ import static org.gradle.util.Matchers.containsText
 class XcodeErrorIntegrationTest extends AbstractXcodeIntegrationSpec {
     @Requires(TestPrecondition.XCODE)
     def "fails to build when project code is broken"() {
-        executer.requireGradleDistribution()
+        executer.requireGradleDistribution().requireOwnGradleUserHomeDir()
```

Comment:
Hmm, I wasn't paying close enough attention before.  It looks like we're actually starting _two_ daemons with these tests.  One to generate the xcode project and another to actually run it.  I made it so we're hackily passing along the GRADLE_USER_HOME, but I think we need to somehow get the full daemon's options (timeout, daemon registry dir, tmpdir, etc) into the Gradle process xcode uses.  Are you seeing the same thing?

Reply:
Let's talk about this.